python-client/tests/test_performance.py

```diff
@@ -233,7 +234,7 @@ def test_diff_accuracy(data, model, threshold, expected_metric, request):
     )
     assert round(result.metric, 2) == expected_metric
     assert type(result.output_df) is bytes
-    assert result.passed
+    assert result.passed == should_pass
```

Comment:
it's worth also asserting messages since we know there should be a message about zero metric